tests/console/rmt/rmt_feature.pm

```diff
@@ -90,7 +100,17 @@ sub run {
         die 'cleanup repos failed';
     }
 
+    record_info('add custom repo');
+    assert_script_run("rmt-cli repos custom add https://download.opensuse.org/repositories/games:/tools/SLE_15_SP3/x86_64/ Games");
+    assert_script_run("rmt-cli repos custom list | grep Games");
+    # attach the custom repo to a product
+    assert_script_run("rmt-cli products enable sle-module-development-tools/15.3/x86_64");
+    assert_script_run("rmt-cli repos custom attach games 2161");
+    assert_script_run("rmt-cli repos custom products games");
+    assert_script_run("rmt-cli repos custom enable games");
```

Comment:
After this line the test finished, to avoid the test data affect later test, it is better to cleanup the test data.

Reply:
I add one line to disable the repo, that's enough for cleanup because I didn't mirror the enabled repos for this custom repo testing.